pkg/pgmodel/end_to_end_tests/main_test.go

```diff
@@ -91,6 +100,12 @@ func withDB(t testing.TB, DBName string, f func(db *pgxpool.Pool, t testing.TB))
 	})
 }
 
+func withSuperuser(t testing.TB, DBName string, f func(db *pgxpool.Pool, t testing.TB)) {
```

Comment:
doesn't seem to be used?

Reply:
must've removed that test. Ah well, it was too brittle anyway